code/backend/batch/utilities/helpers/config/config_helper.py

```diff
@@ -180,13 +184,29 @@ def get_active_config_or_default():
 
     @staticmethod
     def save_config_as_active(config):
+        ConfigHelper.validate_config(config)
         blob_client = AzureBlobStorageClient(container_name=CONFIG_CONTAINER_NAME)
         blob_client = blob_client.upload_file(
             json.dumps(config, indent=2),
             CONFIG_FILE_NAME,
             content_type="application/json",
         )
 
+    @staticmethod
+    def validate_config(config: dict):
+        for document_processor in config.get("document_processors"):
+            document_type = document_processor.get("document_type")
+            unsupported_advanced_image_processing_file_type = (
+                document_type not in ADVANCED_IMAGE_PROCESSING_FILE_TYPES
+            )
+            if (
+                document_processor.get("use_advanced_image_processing")
+                and unsupported_advanced_image_processing_file_type
+            ):
+                raise Exception(
+                    f"Advanced image processing is enabled for document type {document_type}, but only {ADVANCED_IMAGE_PROCESSING_FILE_TYPES} files types are supported."
```

Comment:
sounds confusing ? if the document_type is not supported, why do we say `Advanced image processing is enabled for document type....` ? Is this a typo ?

Reply:
Ah it's because it's been enabled by the user, but it's not supported. It doesn't mean it's been enabled by the app. We'll reword it to try and improve.

Reply:
Renamed slightly, hopefully it's clearer now

Reply:
ahh, I see what you mean. :) It's always tricky to come up with a perfect message.